packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -105,6 +106,19 @@ export class MirrorNodeClient {
             timeout: 10 * 1000
         });
 
+        //@ts-ignore
+        axiosRetry(axiosClient, {
+            retries: parseInt(process.env.MIRROR_NODE_RETRIES!) || 3,
+            retryDelay: () => {
+                const delay = parseInt(process.env.MIRROR_NODE_RETRY_DELAY!) || 500;
+                return delay;
+            },
+            retryCondition: (error) => {
+                return !error?.response?.status || error?.response?.status === 404 || error?.response?.status === 504;
```

Comment:
There might be other codes we care about? For example, maybe retry on 408, 425 and 500 but not 429?

Reply:
Should it even retry on 404, in most cases the mirror node returning 404 is an expected behavior?

Reply:
Yeah 404 retries make sense since Mirror Node may not have yet synced the files with the data.

Reply:
Okay, added array with error codes.